vhdbuilder/packer/vhd-image-builder-mariner-gen2.json

```diff
@@ -0,0 +1,260 @@
+{
+  "variables": {
+    "client_id": "{{env `AZURE_CLIENT_ID`}}",
+    "client_secret": "{{env `AZURE_CLIENT_SECRET`}}",
+    "tenant_id": "{{env `AZURE_TENANT_ID`}}",
+    "subscription_id": "{{env `AZURE_SUBSCRIPTION_ID`}}",
+    "location": "{{env `AZURE_LOCATION`}}",
+    "vm_size": "{{env `AZURE_VM_SIZE`}}",
+    "build_number": "{{env `BUILD_NUMBER`}}",
+    "build_id": "{{env `BUILD_ID`}}",
+    "commit": "{{env `GIT_VERSION`}}",
+    "feature_flags": "{{env `FEATURE_FLAGS`}}",
+    "os_version": "{{env `OS_VERSION`}}",
+    "hyperv_generation": "{{env `HYPERV_GENERATION`}}",
+    "container_runtime": "{{env `CONTAINER_RUNTIME`}}",
+    "teleportd_plugin_download_url": "{{env `TELEPORTD_PLUGIN_DOWNLOAD_URL`}}",
+    "enable_fips": "{{env `ENABLE_FIPS`}}",
+    "imported_sig_image": "{{env `IMPORT_NAME`}}"
+  },
+  "builders": [
+    {
+      "type": "azure-arm",
+      "client_id": "{{user `client_id`}}",
+      "client_secret": "{{user `client_secret`}}",
+      "tenant_id": "{{user `tenant_id`}}",
+      "subscription_id": "{{user `subscription_id`}}",
+      "os_type": "Linux",
+      "os_disk_size_gb": 30,
+      "azure_tags": {
+        "os": "Linux",
+        "now": "{{user `create_time`}}",
+        "createdBy": "aks-vhd-pipeline"
+      },
+      "location": "{{user `location`}}",
+      "vm_size": "{{user `vm_size`}}",
+      "shared_image_gallery": {
+        "subscription": "{{user `subscription_id`}}",
+        "resource_group": "{{user `resource_group_name`}}",
+        "gallery_name": "PackerSigGalleryEastUS",
+        "image_name": "{{user `imported_sig_image`}}",
+        "image_version": "latest"
+      },
+      "managed_image_resource_group_name": "{{user `resource_group_name`}}",
+      "managed_image_name": "CBLMarinerV1Gen2-{{user `create_time`}}",
```

Comment:
s/V1//g 

this should work for all future mariner versions too

Reply:
Replaced with sig_image_name set from init-variables.